src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -207,6 +207,14 @@ public static void FirstOrDefault_OperationCanceledException_PreCanceled(Labeled
         }
 
         [Theory]
+        [MemberData(nameof(UnaryCancelingOperators))]
+        [MemberData(nameof(BinaryCancelingOperators))]
+        public static void ForAll_OperationCanceledException(Labeled<Func<CancellationToken, Operation>> source, Labeled<Func<Action, Operation>> operation)
+        {
+            Functions.AssertEventuallyCanceled((token, canceler) => Cancel(token, canceler, source, operation).ForAll(x => { }));
```

Comment:
Oh, nevermind, I forgot it was Cancel that's using WithCancellation under the covers.  That naming is a bit confusing.  Should Cancel instead be named WithCancellation?